lib/api/src/grpc/proto/points.proto

```diff
@@ -1171,9 +1171,22 @@ message GeoPoint {
 // ---------------------------------------------
 message Usage {
   optional HardwareUsage hardware = 1;
+  optional InferenceUsage inference = 2;
 }
 
 
+// ---------------------------------------------
+// ------------ Inference measurements ----------
+// ---------------------------------------------
+
+message InferenceUsage {
+  map<string, ModelUsage> model = 1;
+}
+
+message ModelUsage {
+  uint64 tokens = 1;
+}
```

Comment:
Is this only intended for text? how will we represent measurements about other modalities like images?